src/aws_request.erl

```diff
@@ -305,210 +168,3 @@ should_retry(#{ type := exponential_with_jitter
   Sleep = Temp div 2 + rand:uniform(Temp div 2),
   timer:sleep(Sleep),
   {ok, RetryState0#{ n => N +1 }}.
-
-%%====================================================================
-%% Unit tests
-%%====================================================================
-
--ifdef(TEST).
```

Comment:
How is the test quality in aws_signature? Can we move some of these tests over?

Reply:
I will double check but just a quick browse shows that most if not all tests that I removed here are covered in `aws_signature`.
Cover coverage report for `aws_signature`:
```
  |------------------------|------------|
  |                module  |  coverage  |
  |------------------------|------------|
  |         aws_signature  |       99%  |
  |   aws_signature_utils  |       90%  |
  |------------------------|------------|
  |                 total  |       97%  |
  |------------------------|------------|
  ```